src/autocomplete/text_completer.js

```diff
@@ -36,9 +37,32 @@ function wordDistance(doc, pos) {
     return wordScores;
 }
 
+function filterStringsFromCompletions(session, pos) {
```

Comment:
since this has a different behaviour to `wordDistance`, I think it would make sense to add a basic test that verifies getting completions works as expected with `filterStringsCompletions` enabled.

Reply:
nit: the name of the function does not really represent what's going on, it returns wordScores for filtered strings so maybe `getWordScoresForFilteredStrings` would better suit it?

Reply:
This name really better suits purpose of function, but could you, please, clarify, which tests you are expecting? Current test is checking scenario with and without `filterStringsCompletions`

Reply:
We only test that with `filterStringsCompletions` enabled comments and doc comments are filtered out, but we don't test that completion works as expected with this option enabled and returns results in expected order (or something like that), since the logic now differs significantly.